File: langs/cowsay.py
import sys
m=" ".join(sys.argv[1:]) or "Hello, World!"
sys.stdout.write(f" {'_'*(len(m)+2)}\n< {m} >\n {'-'*(len(m)+2)}\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n")

File: langs/cowsay.js
const m=process.argv.slice(2).join(" ")||"Hello, World!";
process.stdout.write(` ${"_".repeat(m.length+2)}\n< ${m} >\n ${"-".repeat(m.length+2)}\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n`);
